### PR TITLE
Added test for the requirements §2.1, §3.2 and §3.3

### PR DESCRIPTION
**Pull request by Mikkel-Coder**
Added test for the endpoints: 
- '/v1/api/frontend/login'
- '/v1/api/relay/handshake'

### CODE REVIEW

**Review by Frier03 — APPROVED** [review]
👍🏽

File: Client/client_final.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 from time import sleep
 
 
-BACKEND_URL = 'http://00.00.00.00:8000/v1/api/frontend'
-BACKEND_IP = '00.00.00.00'
+BACKEND_URL = 'http://192.168.137.101:8000/v1/api/frontend'
```
Comment (Frier03) [inline Client/client_final.py:6]: Deja vu?

Comment (Christianm9000) [reply Client/client_final.py:6]: Test on local network

**Review by Christianm9000 — COMMENTED** [review]
all good g

**Pull request merged**